image_width: 800
image_height: 600
camera_name: esp32_cam  # Changed from 'camera' to match expected name
camera_matrix:
  rows: 3
  cols: 3
  data: [800.0, 0.0, 400.0, 0.0, 800.0, 300.0, 0.0, 0.0, 1.0]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.0, 0.0, 0.0, 0.0, 0.0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [800.0, 0.0, 400.0, 0.0, 0.0, 800.0, 300.0, 0.0, 0.0, 0.0, 1.0, 0.0]
